release-calendar/src/client/stylesheets/channelTable.scss

```diff
@@ -0,0 +1,91 @@
+
+button:focus {
```

Comment:
Probably not best to apply styles to all buttons, but use a class or id selector instead.

Reply:
I'm very sure I wont want the outline on any buttons. When the row label used to be a button, I had `outline: none !important;`s everywhere. From what I can tell on the internet, most people do remove it. It's there to make sites more accessible. As long as I check color contrast and html check, hover, before, and after, I should be okay. That said, I can still move it out :)

Reply:
Ok, if you want that to be a site-wide style, move it to the top level stylesheet.